javascript/src/TripDAO.js

```diff
@@ -0,0 +1,3 @@
+export default function findTripsByUser(user) {
```

Comment:
hm, thinking about this again, this may be a bit mean as the only way to create a seam here initially is probably to use some require-rewriting hack like proxyquire or rewire